indico/modules/events/registration/templates/management/import_registrations.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'layout/base.html' %}
+{% from 'forms/_form.html' import form_header, form_rows, form_footer  %}
+{% from 'message_box.html' import message_box %}
+
+{% block content %}
+    {% call message_box('highlight', large_icon=true) %}
+        {% trans num_columns=6 -%}
+            You should upload a CSV (comma-separated values) file with exactly {{num_columns}} columns in the
+            following order:
+            <ul>
+                <li>First Name</li>
+                <li>Last Name</li>
+                <li>Affiliation</li>
+                <li>Position</li>
+                <li>Phone Number</li>
+                <li>E-mail</li>
+            </ul>
+            The fields "First Name", "Last Name" and "E-mail" are mandatory. Users will be matched with existing
+            Indico identities through their e-mail.
+        {%- endtrans %}
+    {% endcall %}
+
+    {{ form_header(form, multipart=True, action=url_for('.registrations_import', regform)) }}
```

Comment:
`true`

Reply:
the url is the same as the one the dialog is loaded from so omitting it should work. also, I think you can use `simple_form()` here

Reply:
Dropzone complains If I specify no URL.